environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/EnvironmentRequest.java

```diff
@@ -79,6 +79,9 @@
     @ApiModelProperty(EnvironmentModelDescription.AWS_PARAMETERS)
     private AwsEnvironmentParameters aws;
 
+    @ApiModelProperty(value = EnvironmentModelDescription.PARENT_ENVIRONMENT_CRN)
+    private String parentEnvironmentCrn;
```

Comment:
can we use name or crn here ?

Reply:
Done in af20f701d8c3e87addc57f612c9b6fc642d0daf5